tools/tertiary-analysis/data/convert-ftype.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="convert-ftype" name="RDS to AnnData converstion" version="v0.0.1+galaxy0">
+  <requirements>
+    <requirement type="package" version="2.3.1">r-seurat</requirement>
+  </requirements>
+  <description>using seurat</description>
+  <command detect_errors="exit_code"><![CDATA[
+ln -s '${input_rds}' input.rds &&
+cp ${script_file} rds2anndata.R &&
+Rscript rds2anndata.R
+]]></command>
+
+  <configfiles>
+    <configfile name="script_file"><![CDATA[
+suppressPackageStartupMessages(library(Seurat))
+rds <- readRDS('input.rds')
+Convert(from=rds, to='anndata', filename='output.h5')
+]]></configfile>
+  </configfiles>
+
+  <inputs>
+    <param name="input_rds" type="data" format="rdata" label="Seurat object in RDS format"/>
```

Comment:
Can we please add some `help=` here in the input param?

Reply:
I've added more description elsewhere, can't think of anything more informative than the label to put in help for this one though.